client/state/privacy-policy/reducer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ *
+ * @format
+ */
+
+/**
+ * Internal dependencies
+ */
+import { combineReducers, createReducer } from 'state/utils';
+import { PRIVACY_POLICY_RECEIVE } from 'state/action-types';
+
+import { privacyPolicySchema } from './schema';
+
+export const entities = createReducer(
+	{},
+	{
+		[ PRIVACY_POLICY_RECEIVE ]: ( state, actions ) => actions.entities,
+	},
+	privacyPolicySchema
+);
+
+export default combineReducers( {
+	entities,
```

Comment:
Instead of dumping the data response we should at least be able to trim down the entities object, which I see only has one key. Do we expect only one private policy, or many?

Reply:
we could expect more versions of the same privacy policy or other types such as TOS it not in the short term though.